src/mmw/mmw/settings/development.py

```diff
@@ -71,3 +71,6 @@
 }
 
 # END TOOLBAR CONFIGURATION
+
+# API key for testing/development
+GOOGLE_MAPS_API_KEY = 'AIzaSyB0D5gjoIHpmy-xdP2cr_0I-E7K6s_L0k4'
```

Comment:
Is this something we shouldn't be checking into the repo and load through env vars instead?